compose-builder/add-security.yml

```diff
@@ -68,11 +68,9 @@ services:
       - security-secrets-setup
 
   security-secrets-setup:
-    image: ${CORE_EDGEX_REPOSITORY}/docker-edgex-secrets-setup-go${ARCH}:${CORE_EDGEX_VERSION}${DEV}
+    image: ${CORE_EDGEX_REPOSITORY}/docker-secrets-setup-go${ARCH}:${CORE_EDGEX_VERSION}${DEV}
```

Comment:
Almost, but missing `security` in the name. Needs to be `docker-security-secrets-setup-go`

Reply:
Done